examples/flutter_gallery/lib/demo/text_field_demo.dart

```diff
@@ -16,13 +16,43 @@ class TextFieldDemo extends StatefulWidget {
 class PersonData {
```

Comment:
@mpcomplete similarly these changes were to actually implement the "SUBMIT" button which previously was dummy. It also makes the lifeStory be saved (previously it wasn't).